crates/miden-lib/src/errors/tx_kernel_errors.rs

```diff
@@ -1,319 +1,305 @@
+use crate::errors::MasmError;
+
 // This file is generated by build.rs, do not modify manually.
 // It is generated by extracting errors from the masm files in the `miden-lib/asm` directory.
 //
 // To add a new error, define a constant in masm of the pattern `const.ERR_<CATEGORY>_...`.
 // Try to fit the error into a pre-existing category if possible (e.g. Account, Prologue,
 // Non-Fungible-Asset, ...).
-//
-// The comment directly above the constant will be interpreted as the error message for that error.
-
-// Transaction Kernel errors are in range 0x20000..0x24000.
-// Its sub categories are:
-// KERNEL is in range 0x20000..0x20040
-// PROLOGUE is in range 0x20040..0x20080
-// EPILOGUE is in range 0x20080..0x200c0
-// TX is in range 0x200c0..0x20100
-// NOTE is in range 0x20100..0x20140
-// ACCOUNT is in range 0x20140..0x20180
-// FOREIGN_ACCOUNT is in range 0x20180..0x201c0
-// FAUCET is in range 0x201c0..0x20200
-// FUNGIBLE_ASSET is in range 0x20200..0x20240
-// NON_FUNGIBLE_ASSET is in range 0x20240..0x20280
-// VAULT is in range 0x20280..0x202c0
 
 // TX KERNEL ERRORS
 // ================================================================================================
 
-/// Provided kernel procedure offset is out of bounds
-pub const ERR_KERNEL_PROCEDURE_OFFSET_OUT_OF_BOUNDS: u32 = 0x20000;
+/// Error Message: "anchor block commitment must not be empty"
+pub const ERR_ACCOUNT_ANCHOR_BLOCK_COMMITMENT_MUST_NOT_BE_EMPTY: MasmError = MasmError::from_static_str("anchor block commitment must not be empty");
```

Comment:
I think the goal with 0.14 was that we wouldn't need to duplicate errors defined in masm and rust (and not have to do any build.rs logic). Although I didn't check everywhere, it seems like a bunch of these are only used in tests. Do we really *need* `MasmError`, or could those strings be moved directly in tests?

Reply:
For example, in the VM, we [hardcode the expected strings](https://github.com/0xMiden/miden-vm/blob/c91fa92d91532c4202cbf1ffff2f9e944747bf68/processor/src/tests.rs#L144-L162) in the tests